tests/inline/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -466,12 +465,16 @@ echo 'bar';
 <p>Test.</p>
 <?php echo 'String.'; ?>
 <p>Test.</p>
-<?php if ($expression == true): ?>
+<?php if (
+    $expression == true
+): ?>
```

Comment:
Somehow, the inline expressions aren't kept one the same line anymore. I didn't look into this in detail yet...